x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -311,6 +311,9 @@ export const endpointMiddlewareFactory: ImmutableMiddlewareFactory<EndpointState
         const response = await coreStart.http.get<HostInfo>(
           resolvePathVariables(HOST_METADATA_GET_ROUTE, { id: selectedEndpoint as string })
         );
+        if (!response) {
+          return;
```

Comment:
Looks like this is silently ignoring a failure? Should we log that this occurred? Should we dispatch an error state that getting details seems to have failed?

Reply:
That would be logical yes. I am not sure what the UX warrants here when this happens though.
@paul-tavares what do you suggest?

Reply:
@ashokaditya why did you need this here?
We have the `catch` down below on line 347 which handles error conditions. `response` here should never be falsey, unless there is a bug in the API that allows it to return nothing as a success `200` http status.

Reply:
Okay, so it turned out that I had to mock some API requests after all. 👍